lighthouse-core/gather/gather-runner.js

```diff
@@ -171,9 +170,12 @@ class GatherRunner {
    * @return {!Promise}
    */
   static beforePass(options, gathererResults) {
+    const blockedUrls = (options.config.blockedUrlPatterns || [])
+      .concat(options.flags.blockedUrlPatterns || []);
```

Comment:
aside: we really need to get the overloaded naming of `options` under control here :)